docs/partials/useCases.md

```diff
@@ -1,5 +1 @@
-The `<auro-breadcrumb>` element should be used in situations where users may:
-
-* ...
-* ...
-* ...
+The `<auro-nav>` element acts as a container for a series of `<auro-hyperlinks>`. The `<auro-nav>` element should only be used for multi-page projects that have 3 to 5 levels of page depth. Additionally, `<auro-nav>` should only display in the UI on page depth 3 or above.
```

Comment:
```suggestion
The `<auro-nav>` element acts as a container for a series of `<auro-hyperlinks>`. The `<auro-nav>` element should only be used for multi-page projects that have 3 to 5 levels of page depth.
```

The second sentence already states what the third sentence repeats. Less words == more understanding.